service/docs/source/overview.rst

```diff
@@ -819,13 +828,11 @@ showcases two optional features of the GEOPM Runtime:
    it to 200 milliseconds, up from the default 5 milliseconds, results in
    approximately 50 rows of samples in the trace file (calculated as five
    samples per second over ten seconds).
-3. **Specify Invocation Name**: The optional ``GEOPM_PROGRAM_FILTER``
-   environment variable allows you to explicitly list program
-   invovation names that will request profiling, other programs will
-   not be affected by ``LD_PRELOAD`` of ``libgeopm.so``.  For this
-   reason a user will typically set these two environment variables
-   together.  This is especially important when profiling programs
-   within a bash script.
+3. **Disable Network Use** The ``GEOPM_CTL_LOCAL`` environment
+   variable may be set which disables all intra-node communication
+   between the controllers on each node, thereby generating a unique
+   report file per host node over which the application processes are
+   launched.
```

Comment:
Also note that when MPI is disabled in the runtime, GEOPM_CTL_LOCAL is used by default.  This is a change in this PR.